packages/discord-bot/src/discord/commands/setEthAddress.ts

```diff
@@ -47,14 +52,21 @@ export abstract class SetEthAddress {
 
     const account = manager.ledger.account(baseIdentityId);
 
-    const existing = account.identity.aliases.find((alias) => {
+    const existingEthAliases = account.identity.aliases.filter((alias) => {
       const parts = sc.core.graph.NodeAddress.toParts(alias.address);
       return parts.indexOf('ethereum') > 0;
     });
 
-    if (existing) {
+    const latestEthAlias = existingEthAliases[existingEthAliases.length - 1];
```

Comment:
```suggestion
    const [latestEthAlias] = existingEthAliases.slice(-1);
```

Reply:
More concise, yes, more readable, probably not.